### ISSUE TITLE
SQS create-queue cli command is throwing error "QueueAlreadyExists" when trying to create a queue with same name as existing queue.

### ISSUE THREAD

**Issue by rajatm077** [localstack/localstack#3035]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description
SQS create-queue command is throwing error "QueueAlreadyExists" when trying to create a queue with same name as existing queue.
aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-south-1 --queue-name myqueue.fifo
...

## Expected behavior
It should return the url of existing queue.
...

## Actual behavior
If we execute this command for the first time, it'll create the queue but on running it again, it'll throw an error: QueueAlreadyExists.
aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-south-1 --queue-name myqueue.fifo
...

# Steps to reproduce
Run this command twice:
aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-south-1 --queue-name myqueue.fifo

## Command used to start LocalStack
localstack start
...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
...

localstack info:
localstack/localstack    latest              6fd3f7511cde        18 hours ago        668MB

**Comment by usmangani1** [comment]
@rajatm077 Which version of localstack are you using ?
I am not able to reproduce this issue on version 0.11.5.
```
localstack$ aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-south-1 --queue-name myqueue.fifo
{
    "QueueUrl": "http://localhost:4566/000000000000/myqueue.fifo"
}

localstack$ aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-south-1 --queue-name myqueue.fifo
{
    "QueueUrl": "http://localhost:4566/000000000000/myqueue.fifo"
}
```
Please let us know which version of localstack are you using currently.

**Comment by rajatm077** [comment]
Sorry for providing incomplete and inaccurate details. Let me explain the complete scenario.
1. Launch localstack: DEFAULT_REGION=ap-southeast-1 localstack start
2. LocalStack version: 0.11.5
```
$ aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-southeast-1 --queue-name myqueue.fifo --attributes FifoQueue=true
{
    "QueueUrl": "http://localhost:4566/000000000000/myqueue.fifo"
}

$ aws --endpoint-url=http://localhost:4566 sqs set-queue-attributes --queue-url http://localhost:4566/000000000000/myqueue.fifo --attributes VisibilityTimeout=5600

$ aws --endpoint-url=http://localhost:4566 sqs create-queue --region ap-southeast-1 --queue-name myqueue.fifo --attributes FifoQueue=true
An error occurred (QueueAlreadyExists) when calling the CreateQueue operation: The specified queue already exists.
```

What I'm trying to do is on my application startup, I'm creating queues and setting attributes. So if the queue exists it returns the queue url and sets the attributes but when I integrate my code with localstack for local testing it is throwing error.
<img width="1536" alt="Screenshot 2020-09-28 at 5 28 27 PM" src="https://user-images.githubusercontent.com/10363887/94429509-07b36000-01b0-11eb-8720-3bbdeb4a0311.png">

Desired behaviour:

```
$ aws sqs create-queue --region ap-southeast-1 --queue-name myqueue.fifo --attributes FifoQueue=true
{
    "QueueUrl": "https://sqs.ap-southeast-1.amazonaws.com/******/myqueue.fifo"
}
$ aws sqs set-queue-attributes --queue-url https://sqs.ap-southeast-1.amazonaws.com/******/myqueue.fifo --attributes VisibilityTimeout=5600
$ aws sqs create-queue --region ap-southeast-1 --queue-name myqueue.fifo --attributes FifoQueue=true
{
    "QueueUrl": "https://sqs.ap-southeast-1.amazonaws.com/******/myqueue.fifo"
}
```
<img width="1536" alt="Screenshot 2020-09-28 at 5 32 47 PM" src="https://user-images.githubusercontent.com/10363887/94430108-00d91d00-01b1-11eb-9f9f-dd4e598e2f8e.png">

**Comment by whummer** [comment]
Thanks for the update @rajatm077 . This should be fixed upstream in spulec/moto#3345 and is now available in the `latest` LocalStack version. Can you please pull the Docker image and give it another try? Please report here if the problem persists. Thanks!

**Comment by rajatm077** [comment]
@whummer @usmangani1 Yeah, it is working now. Thanks for fixing it.
